lib/gradebook.rb

```diff
@@ -1 +1,41 @@
+class Gradebook
+  attr_reader :instructor,
+              :courses
 
+  def initialize(instructor)
+    @instructor = instructor
+    @courses = []
+  end
+
+  def add_course(course)
+    @courses << course
+    @courses
```

Comment:
I'm pretty sure an implicit return of line 11 would also work:
```suggestion
def add_course(course)
    @courses << course
end
```